sdcm/tester.py

```diff
@@ -1807,6 +1807,8 @@ def run_stress_cassandra_thread(
         else:
             timeout = get_timeout_from_stress_cmd(stress_cmd) or self.get_duration(duration)
 
+        if "no-warmup" not in stress_cmd:
+            stress_cmd += " no-warmup"
```

Comment:
It's plain like that ? It shouldn't be part of a specific group of parameters ?

Reply:
your'e right it's not, need to put it other place

Reply:
fixed